sdk/eventhub/event-hubs/review/event-hubs.api.md

```diff
@@ -82,6 +91,29 @@ export interface EventDataBatch {
     tryAdd(eventData: EventData | AmqpAnnotatedMessage, options?: TryAddOptions): boolean;
 }
 
+// @public
+export class EventHubBufferedProducerClient {
```

Comment:
Would this come to service-bus too?

Reply:
We haven't talked about doing this for service-bus as well yet. Some of the reasons we're doing this for event hubs (e.g. idempotent producer) don't apply to service bus, but service bus is definitely out of scope :smile: